tasks/components/OrganizationForm.tsx

```diff
@@ -97,7 +97,7 @@ export const OrganizationForm = ({
   const inputClasses = tw('mt-1 block rounded-md w-full border-gray-300 shadow-sm focus:outline-none focus:border-hw-primary-500 focus:ring-hw-primary-500')
 
   function validateShortName(organization: OrganizationGeneralInfoDTO) {
-    if (organization.shortName === '') {
+    if (organization.shortName.trim() === '') {
```

Comment:
It would be smart if we move `organization.shortName` into a separate variable. This variable should be then used throughout the rest of the function. This applies for all `validate...()` functions.